src/commands/coin/coinCheck.ts

```diff
@@ -0,0 +1,24 @@
+import { Message } from 'discord.js';
+import { Args, Command, CommandOptions } from '@sapphire/framework';
+import { ApplyOptions } from '@sapphire/decorators';
+import { getCoinBalanceByUserId } from '../../components/coin';
+import { BOT_PREFIX } from '../../bot';
+
+@ApplyOptions<CommandOptions>({
+  aliases: ['cc', 'coin-balance-check', 'coin-check'],
+  description: "Checks a user's Codey coin balance.",
+  detailedDescription: `**Examples:**\n\`${BOT_PREFIX}coin-check @Codey\`\n\`${BOT_PREFIX}cc @Codey\``,
+  requiredUserPermissions: ['ADMINISTRATOR']
```

Comment:
Why does this require admin? We're displaying user coins on their profile as well so I think this command should be available to everyone?

Reply:
It was discussed during a meeting in the previous term that a user may not want their coin balance to be checked by other people, so this command was made admin-only. If someone wants to check someone else's balance, then they would have to ask the person to do `.bal` in a public channel.
I did not realize that coin balance would be displayed on a user profile. Are we sure that this is a change that is wanted? @marko-polo-cheno

Reply:
@smilesthesad @Picowchew good point
imo we should open it up to public, but let's talk abt it on Wed.
if public, then we can have a leaderboard command :o